.github/30-build-installation-issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Build/Installation Issue
+about: Use this template for build/installation issues
+
+---
+
+<em>tag:build_template</em>
+
+**Please provide the following information**
+
+**System information**
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04) :
+- TensorBoard version :
+- TensorFlow version(if different from TensorBoard) :
```

Comment:
nit: space between "version" and "(if"

Also a nit, but no spaces before the colons on this line and the two above

Reply:
Done